src/main/java/org/openstreetmap/atlas/geography/atlas/command/PBFToAtlasSubCommand.java

```diff
@@ -0,0 +1,214 @@
+package org.openstreetmap.atlas.geography.atlas.command;
+
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.commons.io.FilenameUtils;
+import org.openstreetmap.atlas.geography.MultiPolygon;
+import org.openstreetmap.atlas.geography.atlas.pbf.AtlasLoadingOption;
+import org.openstreetmap.atlas.geography.atlas.pbf.OsmPbfLoader;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.tags.filters.ConfiguredTaggableFilter;
+import org.openstreetmap.atlas.utilities.configuration.StandardConfiguration;
+import org.openstreetmap.atlas.utilities.runtime.Command;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+import org.openstreetmap.atlas.utilities.runtime.FlexibleSubCommand;
+
+/**
+ * This command converts an OSM PBF file to an Atlas file. It requires the path to a pbf and an
+ * output file. It also takes a number of optional parameters.
+ *
+ * @author bbreithaupt
+ */
+public class PBFToAtlasSubCommand implements FlexibleSubCommand
+{
+    private static final String NAME = "pbf-to-atlas";
+    private static final String DESCRIPTION = "Converts a PBF to an Atlas file.";
+
+    // Required parameters
+    private static final Command.Switch<File> INPUT_PARAMETER = new Command.Switch<>("pbf",
```

Comment:
If you import `Switch` above, you wont have to call `Command.` every time you want to access the class